config/jobs/kubernetes-sigs/release-sdk/release-sdk-presubmits.yaml

```diff
@@ -6,14 +6,21 @@ presubmits:
     path_alias: "sigs.k8s.io/release-sdk"
     spec:
       containers:
-      - image: gcr.io/k8s-staging-releng/releng-ci:latest-go1.17
+      - image: gcr.io/k8s-staging-releng/k8s-ci-builder:latest-default
         imagePullPolicy: Always
         command:
-        - go
+        - wrapper.sh
         args:
+        - go
         - run
         - mage.go
         - Test
+        env:
+          - name: DOCKER_IN_DOCKER_ENABLED
+            value: "true"
```

Comment:
I would prefer using the preset label instead:
```yaml
    labels:
      preset-dind-enabled: "true"
```

Reply:
@saschagrunert Fixed, PTAL. I wasn't aware that we have this preset.